source/software/r_package_management.rst

```diff
@@ -6,51 +6,106 @@ R package management
+
+.. code-block:: bash
+
+      $ mkdir -p ${VSC_DATA}/Rlibs/rocky8/icelake/R-4.2.2
+
+The next step is to ensure such install locations are used by default in the R package installation process.
+This can be done by setting the `R_LIBS_USER` variable to in the `~/.Renviron` file as follows:
+
+.. code-block:: bash
+
+      $ echo 'R_LIBS_USER=${VSC_DATA}/Rlibs/${VSC_OS_LOCAL}/${VSC_ARCH_LOCAL}/R-${EBVERSIONR}' >> ~/.Renviron
+
+The `${VSC_OS_LOCAL}` and `${VSC_ARCH_LOCAL}` environment variables are predefined
+and match the OS version (e.g. `rocky8`) and CPU model (e.g. `icelake`) of the node.
+The `${EBVERSIONR}` variable contains the R version (e.g. `4.2.2`) of the currently loaded
+R module.
+
+R will now use this path as default install path, ensuring you are always installing
+your packages in the appropriate R library folder.
 
-#. Load the appropriate R module, i.e., the one you want the R package
-   to be available for::
+.. note::
+
+  This `.Renviron` configuration will also work as expected in Open OnDemand apps
+  such as RStudio Server.
+
+The next step is to load the appropriate R module and launch R.
+
+.. code-block:: bash
+
+      $ module load R/4.2.2-foss-2022b
+      $ R
+
+From here, installing packages can be as simple as:
+
+.. code-block:: r
 
-      $ module load R/3.2.1-foss-2014a-x11-tcl
+      > install.packages("DEoptim")
 
-#. Start R and install the package (preferably in your $VSC_DATA directory)::
 
-      > install.packages("DEoptim", lib="/data/leuven/304/vsc30468/R/")
+If you are unsure whether R will install in the correct location, you can first list
+the known library locations by executing `.libPaths()`. The first location is the
+default one.
 
-#. Alternatively you can download the desired package::
+You can also specify your desired library path as an extra argument in the install command.
+This will take precedent over any defaults.
+
+.. code-block:: r
+
+      > Rlibs <- "/path/to/my/R_library"
+      > install.packages("DEoptim", lib = Rlibs)
+
+Alternatively you can download the desired package
+
+.. code-block:: bash
 
       $ wget cran.r-project.org/src/contrib/Archive/DEoptim/DEoptim_2.0-0.tar.gz
 
-      and install it with::
+and install it from the command line with
+
+.. code-block:: bash
   
-      $ R CMD INSTALL DEoptim_2.0-0.tar.gz  -l /$VSC_DATA/R/
-      
-#. These packages might depend on the specific R version, so you may
-   need to reinstall them for the other version.
-   
-Some R packages depend on libraries installed on the system.  In that case,
-you first have to load the modules for these libraries, and only then proceed
-to the R package installation.  For instance, if you would like to install
-the `gsl` R package, you would first have to load the module for the GSL
-library, .e.g., ::
-
-   $ module load GSL/2.5-GCC-6.4.0-2.28
+      $ R CMD INSTALL DEoptim_2.0-0.tar.gz  -l ${VSC_DATA}/Rlibs/rocky8/icelake/R-4.2.2
```

Comment:
For consistency, shall we also recommend using `VSC_*` env vars instead of hard-coded path?

Reply:
Yes I will do that with a note above that it should be done from a compute node specifically.